packages/launch/package.json

```
@@ -38,7 +38,8 @@
 		"@wordpress/core-data": "^2.16.0",
 		"@wordpress/icons": "^2.4.0",
 		"@wordpress/url": "^2.17.0",
-		"classnames": "^2.2.6"
+		"classnames": "^2.2.6",
+		"react-router-dom": "^5.2.0"
```

By using the existing package version from `client` I guess we can get rid of the [yarn.lock noise](https://github.com/Automattic/wp-calypso/pull/47062/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de) in this PR.

```suggestion
		"react-router-dom": "^5.1.2"
```

---

Downgraded to `5.1.2`. This PR doesn't affect anymore `yarn.lock`